pkg/chunk/aws/s3_storage_client.go

```diff
@@ -155,7 +155,7 @@ func (a s3ObjectClient) bucketFromKey(key string) string {
 	}
 
 	hasher := fnv.New32a()
-	hasher.Write([]byte(key))
+	_, _ = hasher.Write([]byte(key))
```

Comment:
Could add `hash.Hash32.Write` to the project `.errcheck-exclude`

Reply:
@zendern May you do it, please?

Reply:
@jtlisi and @pracucci ok'd this so going to mark all the `hash.Hash32.Write to the project .errcheck-exclude` as resolved since it a limitation to golangci-lint until the errcheck version is updated. I did open this to address it https://github.com/golangci/golangci-lint/issues/959. So hopefully I or someone can get around to fixing that.